docs/05-internals/design/adr/ADR-0007-zio-fication-of-responders.md

```diff
@@ -0,0 +1,163 @@
+# ADR-0007 ZIO-fication of Responders
+
+Date: 2023-02-20
+
+## Status
+
+Draft
+
+## Context
+
+In order to remove all _Akka_ dependencies, we have to migrate the existing `Responder`s to a `ZIO` based
+implementation.
+This migration should be possible to do on a per `Responder` basis so that we do not a single "big-bang" release with
```

Comment:
`so that we do not a` -> `so that we do not do a`

Reply:
=> 955527e56707acb6809391219c7311ba1925edbc